internal/xds/rbac/converter.go

```diff
@@ -72,10 +76,24 @@ func getCustomConfig(config *anypb.Any) (json.RawMessage, string, error) {
 			return nil, "", fmt.Errorf("failed to unmarshal resource: %v", err)
 		}
 		return convertCustomConfig(typedStruct.TypeUrl, typedStruct.Value)
+	case stdoutType:
+		stdoutLoggerConfig := &v3auditloggersstreampb.StdoutAuditLog{}
+		if err := config.UnmarshalTo(stdoutLoggerConfig); err != nil {
+			return nil, "", fmt.Errorf("failed to unmarshal resource: %v", err)
+		}
+		return convertStdoutConfig(stdoutLoggerConfig)
 	}
 	return nil, "", fmt.Errorf("custom config not implemented for type [%v]", config.GetTypeUrl())
 }
 
+func convertStdoutConfig(config *v3auditloggersstreampb.StdoutAuditLog) (json.RawMessage, string, error) {
+	json, err := protojson.Marshal(config)
+	if err != nil {
+		return nil, "", err
+	}
+	return json, stdout.Name, nil
```

Comment:
Seems like there is an check for non nil error at the callsite of `getCustomConfig` and the returned json.RawMessgage and string is anyway unused, can we skip the non-nil error check from this method and rely on the callsite check? 

what i mean is:

```suggestion
	json, err := protojson.Marshal(config)
	return json, stdout.Name, err
```

Reply:
Done